src/shim/sdk/UserSDK.ts

```diff
@@ -18,6 +18,20 @@ export function changePasscode(currentPasscode: string, newPasscode: string) {
         .map(() => undefined)
         .toPromise();
 }
+/**
+ * Rotates the user current private key.
+ * @param {string} passcode
```

Comment:
You should add a comment about what this parameter is

Reply:
done